content/_code-samples/freeze/py/freeze_token.py

```diff
@@ -0,0 +1,38 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import IssuedCurrencyAmount, TrustSet
+from xrpl.models.transactions import TrustSetFlag
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # Connect to testnet
+
+token_name = "FOO"
+
+# Amount a trustline can handle, for this transaction it is set to 0
+value = "100"
+
+target_addr = generate_faucet_wallet(client=client).classic_address
+
+sender_wallet = generate_faucet_wallet(client=client)
+
+# Build trustline freeze transaction
+trustset = TrustSet(account=sender_wallet.classic_address, limit_amount=IssuedCurrencyAmount(
+    currency= token_name,
+    issuer=target_addr,
+    value = value),
+    flags=TrustSetFlag.TF_SET_FREEZE) 
+
+# Sign transaction
+stxn = safe_sign_and_autofill_transaction(trustset, sender_wallet, client)
+
+# Submit transaction and wait for result
```

Comment:
Same here: 
Ex. 
```
print("Generated wallet, now setting a trustline with the TF_SET_FREEZE flag enabled...")
```

Reply:
All done. What caused the confusion was, I split the messages to show exactly where an action is carried out
<img width="602" alt="Screenshot 2022-11-17 062006" src="https://user-images.githubusercontent.com/47371105/202364900-a319b00f-41e2-47b5-b8b8-139bab396436.png">

Reply:
Makes sense! Sorry for my confusion :)